apps/daimo-mobile/src/sync/sync.ts

```diff
@@ -335,15 +336,17 @@ function addNamedAccounts(old: EAccount[], found: EAccount[]): EAccount[] {
 
 /** Add transfers based on new Transfer event logs */
 function addTransfers(
-  old: TransferClog[],
-  logs: TransferClog[]
+  oldLogs: TransferClog[],
+  newLogs: TransferClog[]
 ): TransferClog[] {
-  // Sort new logs
+  const { logs, remaining } = addLandlineTransfers(oldLogs, newLogs);
+
+  logs.push(...remaining);
+
+  // Sort logs
   logs.sort((a, b) => {
-    if (a.blockNumber !== b.blockNumber) return a.blockNumber! - b.blockNumber!;
-    return a.logIndex! - b.logIndex!;
+    return a.timestamp - b.timestamp;
```

Comment:
👍 

nit, but transfers that occur in the same block might switch places in the history. 

could do something like

```
let diff = a.timestamp - b.timestamp;
if (diff !== 0) return diff;
return (a.logIndex || 0) - (b.logIndex || 0);
```